src/graph/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 file(GLOB_RECURSE graph_src *.cpp)
-set(GRAPH_SRC "${graph_src}" PARENT_SCOPE)
+add_library(graph_lib STATIC "${GRAPH_HEADERS}" "${graph_src}")
```

Comment:
maybe I don't understand, where files for `${GRAPH_HEADERS}`

Reply:
You're right, variable name definition was different. Also, header vars were empty, so I have reorganized header traversal as well